elasticsearch-monitoring/templates/prometheusrule.yaml

```diff
@@ -31,6 +32,7 @@ spec:
       annotations:
         description: 'The Elasticsearch Cloudwatch metrics exporter for {{`{{ $labels.job }}`}} is down!'
         summary: Elasticsearch monitoring is DOWN!
```

Comment:
I'd update the summary here. It's not exactly monitoring IMO

Reply:
It's not all the ES monitoring, but it's part of it. I think it's ok like this, as the alert is clarified in the `description` annotation.